content/pt-br/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Aplicando Padrões de Segurança de Pod com `Labels` em Namespace
+reviewers:
+- tallclair
+- liggitt
+content_type: task
+min-kubernetes-server-version: v1.22
+---
+
+Os namespaces podem ser rotulados para aplicar os [Padrões de segurança de pod]
+(/docs/concepts/security/pod-security-standards). As três políticas
+[privilegiado](/docs/concepts/security/pod-security-standards/#privileged), 
+[linha de base](/docs/concepts/security/pod-security-standards/#baseline)
+e [restrito](/docs/concepts/security/pod-security-standards/#restricted) 
+cobrem amplamente o espectro de segurança e são implementados pela 
+[segurança de Pod](/docs/concepts/security/pod-security-admission/) 
+{{< glossary_tooltip text="controlador de admissão" term_id="admission-controller" >}}.
+
+## {{% heading "prerequisites" %}}
+
+{{% version-check %}}
+
+- Garantir que a `PodSecurity` do [portal de funcionalidades]
+(/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features) 
```

Comment:
This link "portal de funcionalidades" is broken. There is a space between backspace and parenthesis.